src/SDWaveFile.cpp

```diff
@@ -173,14 +173,48 @@ int SDWaveFile::begin()
   return 1;
 }
 
+int SDWaveFile::findData(int offset) {
+  uint8_t byteread;
+  int count = 0, fileSize;
+  uint32_t data;
+  uint8_t buffer[] = {0, 0, 0, 0};
+
+  fileSize = _file.size() - offset;
+
+  //check if "data" field is present
+  while (count < fileSize) {
```

Comment:
Another option is to use: `while (_file.available())` here, then we don't need the file size calculated and the offset passed in.

Reply:
ok is better save space and remove a variable, i'm agree with you